src/Server.php

```diff
@@ -41,6 +44,9 @@ public function __construct(SocketServerInterface $io)
                     $conn->emit('resume');
                 });
             });
+            $parser->on('error', function (\Exception $e) {
+                $this->emit('error', [$e]);
```

Comment:
How is this related to your above change? At the very least, this needs some documentation :+1: